src/firebase/firebaseDatabase.ts

```diff
@@ -0,0 +1,12 @@
+import { db } from "../firebase/firebase";
+import { setDoc, doc } from "firebase/firestore";
+import { IClient } from "../types/client";
+
+export const setFirestoreDatabaseById = async (client: IClient) => {
+  const cityRef = doc(db, "clients", `${client.uid}`);
+  await setDoc(cityRef, {
+    name: client.name,
+    email: client.email,
+    role: "user",
+  });
+};
```

Comment:
example:
```suggestion
export const setFirestoreDatabaseById = async (collection: string, id: string, data: any) => {
  const cityRef = doc(db, collection, id);
  await setDoc(cityRef, data);
};
```
typs needs to be improved